actionpack/test/controller/base_test.rb

```diff
@@ -172,14 +172,12 @@ def test_process_should_be_precise
     assert_equal "The action 'non_existent' could not be found for EmptyController", exception.message
   end
 
-  if defined?(DidYouMean) && DidYouMean.respond_to?(:correct_error)
-    def test_exceptions_have_suggestions_for_fix
-      use_controller SimpleController
-      exception = assert_raise AbstractController::ActionNotFound do
-        get :non_existent
-      end
-      assert_match "Did you mean?", exception.message
+  def test_exceptions_have_suggestions_for_fix
+    use_controller SimpleController
+    exception = assert_raise AbstractController::ActionNotFound do
+      get :ello
```

Comment:
Just curious why changed some words in the commit?

`non_existent` -> `ello`
`magazine` -> `boko`
`title` -> `naem`
`not_found` -> `ello`
`missing_param_key` -> `invalid_param_key`
`monkeys` -> `taggingz`
`kateggorisatons` -> `kategorisation`

Reply:
Sorry, I should have mentioned this.
The `SpellChecker` has a treshold for corrections:
https://github.com/ruby/did_you_mean/blob/master/lib/did_you_mean/spell_checker.rb#L14
If there is not enough similarity it won't return it as a suggestion.

So `non_existent` does not meet the treshold for `hello`.
```ruby
DidYouMean::SpellChecker.new(dictionary: %w[hello]).correct('non_existent')
=> []
DidYouMean::SpellChecker.new(dictionary: %w[hello]).correct('ello')
=> ["hello"]
```

The treshold makes sense for spelling errors.
But I personally think it helps to get a suggestion even if there is little overlap.
For example for 'title' and 'name' or when the known valid options are small (like in controller actions or model attributes).